### PR TITLE
Set up the application

### PR DESCRIPTION
**Pull request by ClaudiaLapalme**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
 Lost on campus? Pikaroute will help you pick a route (SOEN 390 Project)
+
+## Set up
+
+### Installations
+
+1. Install GitBash
+2. Install Node.js and / or update to the latest version
+3. `npm install -g ionic` 
+4. npm install @ionic/angular@latest --save
```
Comment (aiceball) [inline README.md]: Should we lock onto a version?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
 # pikaroute
 Lost on campus? Pikaroute will help you pick a route (SOEN 390 Project)
+
+## Set up
+
+### Installations
+
+1. Install GitBash
```
Comment (aiceball) [inline README.md]: Should we have a "useful tools" or smthn instead? Because I don't see GitBash as part of the product install

Comment (danseremet) [reply README.md]: 1. Install good terminal.

Comment (ClaudiaLapalme) [reply README.md]: any terminal is fine, I can edit it. I should probably also say that these are the instructions for Windows only

Comment (danseremet) [reply README.md]: I can say they work on linux as well

File: README.md (hunk 1 above)
Comment (ClaudiaLapalme) [reply README.md]: I think that we should always be on the latest version and it is up to the developers to update

**Pull request merged**